src/main/java/de/tum/in/test/api/ast/type/FunctionType.java

```diff
@@ -0,0 +1,51 @@
+package de.tum.in.test.api.ast.type;
+
+import com.github.javaparser.ast.Node;
+import com.github.javaparser.ast.stmt.ExplicitConstructorInvocationStmt;
+import com.github.javaparser.ast.stmt.ReturnStmt;
+import com.github.javaparser.utils.Pair;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public enum FunctionType {
```

Comment:
What would be the use case here? I cannot imagine when this would be useful.
If there is one, it should be thoroughly documented in the Javadoc of the instances.

Reply:
[Implemented part of the feedback](https://github.com/ls1intum/Ares/pull/248/commits/7ed9fe2ac7f61d955c71ff99f56e550f8be89531): Removed `FunctionType`